.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "third_party/warp_transducer/submodule"]
+	path = third_party/transducer/submodule
+	url = https://github.com/HawkAaron/warp-transducer
+	branch = master
```

Comment:
Could you add `ignore = dirty` so that whatever the intermediate files in this directory would be ignored by git.

Also I am wondering the effect of `branch = master`. What happens if there is a new commit in the master branch?
Typically, it's not necessary to add branch.
https://github.com/pytorch/pytorch/blob/master/.gitmodules

Reply:
This is about the build files etc., right? We should probably rather build outside of the submodule in third_party/transducer, if that's the only reason. Otherwise, we should be able to just standardize on PyTorch and add this.